CLAUDE.md

```
@@ -195,11 +195,21 @@ User::factory()->create(['is_active' => true, 'email_verified_at' => now()])
 
 ### DB for tests
 
-Tests need a DB. Production CI uses MariaDB 11. For local dev without MySQL, set in `.env.testing`:
+Tests need a real MariaDB DB — matching CI (MariaDB 11) — not SQLite. SQLite's lenient identifier
+quoting has silently masked real bugs before (e.g. `->latest()` defaulting to a nonexistent
+`created_at` column on `$timestamps = false` models passed locally, failed on CI). Run via the
+`cli` compose service, which points at the stack's `db` service automatically:
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)